code/_onclick/hud/alert.dm

```diff
@@ -256,6 +258,74 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	icon_state = "blobbernaut_nofactory"
 	alerttooltipstyle = "blob"
 
+// BLOODCULT
+
+/obj/screen/alert/bloodsense
+	name = "Blood Sense"
+	desc = "Allows you to sense blood that is manipulated by dark magicks."
+	icon_state = "cult_sense"
+	var/image/finder
+	var/image/sacimage
+	var/mob/sacmob
+
+/obj/screen/alert/bloodsense/Initialize()
+	..()
+	finder = new('icons/mob/screen_alert.dmi', "no_sacrifice")
+	var/datum/mind/sacmind = GLOB.sac_target
+	sacmob = sacmind.current
+	var/datum/job/sacjob = SSjob.GetJob(sacmind.assigned_role)
+	var/datum/preferences/sacpref = sacmob.client.prefs
+	var/icon/reshape = get_flat_human_icon(null, sacjob, sacpref)
+	reshape.Shift(SOUTH, 4)
+	reshape.Crop(7,4,26,30)
+	reshape.Crop(-5,-2,26,29)
+	sacimage = reshape
+	add_overlay(finder)
+	START_PROCESSING(SSprocessing, src)
+	process()
```

Comment:
You shouldn't need to process once here, just leave it for the ticker to process